pseudopotential_calculator/calculations/slab.py

```diff
@@ -34,6 +34,45 @@ def kpoint_mp_grid(self: Self) -> str:
         return f"{self.n_k_points} {self.n_k_points} {1}"
 
 
+def repeat_slab(slab: Atoms, amount: tuple[int, int, int]) -> Atoms:
+    return slab.repeat(amount)  # type: ignore bad lib
+
+
+def add_atom_onto_slab(atom: Atom, slab: Atoms) -> Atoms:
+    slab.append(atom)  # type: ignore bad lib
+    return slab
+
+
+def get_top_position(atom: Atoms) -> list[float]:
+    positions = atom.positions  # type: ignore bad lib
+    return max(positions, key=lambda x: x[2])  # type: ignore bad lib
```

Comment:
better to do np.max(positions[:, 2]), but in this case re-use get_sorted_heights